crates/rpc-types/src/eth/transaction/mod.rs

```diff
@@ -315,4 +317,24 @@ mod tests {
         let deserialized: Transaction = serde_json::from_str(&serialized).unwrap();
         assert_eq!(transaction, deserialized);
     }
+
+    #[test]
+    fn serde_minimal_transaction() {
+        let transaction = Transaction {
+            hash: B256::with_last_byte(1),
+            nonce: 2,
+            from: Address::with_last_byte(6),
+            value: U256::from(8),
+            gas: U256::from(10),
+            input: Bytes::from(vec![11, 12, 13]),
+            ..Default::default()
+        };
+        let serialized = serde_json::to_string(&transaction).unwrap();
+        assert_eq!(
+            serialized,
+            r#"{"hash":"0x0000000000000000000000000000000000000000000000000000000000000001","nonce":"0x2","blockHash":null,"blockNumber":null,"transactionIndex":null,"from":"0x0000000000000000000000000000000000000006","to":null,"value":"0x8","gas":"0xa","input":"0x0b0c0d"}"#
```

Comment:
where is this fixture from

Reply:
that's just a fixture from the test above with all options set to None, the idea was to test that we can handle any of the optional fields missing

I can change it to a valid legacy tx payload so we avoid unexpected test failures when modifying `Transaction` in the future